tests/spread/general/overlay-logs/task.yaml

```
@@ -0,0 +1,7 @@
+summary: Check that actions done inside the overlay are logged
+
+execute: |
+  error_pattern="E: Unable to locate package idontexist"
+  rockcraft --verbosity=debug 2>&1 >/dev/null | MATCH "$error_pattern"
+  rockcraft_log_file=$(find /root/.local/state/rockcraft/log/ -name 'rockcraft*.log' | sort -n | tail -n1)
+  cat "$rockcraft_log_file" | MATCH "$error_pattern"
```

Hi there, your local neighbourhood busybody here to let you know that `MATCH` actually runs `grep -q -E` [under the hood](https://github.com/snapcore/spread/blob/ded9133cdbceaf01f8a1c9decf6ff9ea56e194d6/spread/client.go#L790).

```suggestion
  MATCH "$error_pattern" "$rockcraft_log_file"
```

I don't have anything valuable to add. Just saw the lint failure while I was trying to find an unrelated build failure

---

oh of course! I have a blind spot with `MATCH` sometimes